vispy/visuals/visual.py

```diff
@@ -30,6 +30,11 @@ class Visual(object):
     bounds_change : Event
         Emitted when the bounds of the visual have changed.
 
+    Notes
```

Comment:
This is not accurate, as it is possible to use the Visual classes without the scenegraph.